packages/webapp/src/elections/components/ongoing-election-components/meeting-link/meeting-link-modal.tsx

```diff
@@ -70,23 +75,36 @@ const ModalStepZoom = ({ close }: ModalStepProps) => {
 };
 
 interface ModalStepGetLinkProps extends ModalStepProps {
-    requestMeetingLink: (throwOnError: boolean) => Promise<void>;
+    requestMeetingLink: () => Promise<void>;
+    submitMeetingLink: (meetingLink: string) => Promise<void>;
     stage: RoundStage;
 }
 
 const ModalStepGetLink = ({
     close,
     requestMeetingLink,
+    submitMeetingLink,
     stage,
 }: ModalStepGetLinkProps) => {
     const [isSuccess, setIsSuccess] = useState(false);
     const [isLoading, setIsLoading] = useState(false);
-    const onContinue = async () => {
+    const [fields, setFields] = useFormFields({
+        meetingLink: "",
+    });
+
+    const onContinue = async (e: React.FormEvent) => {
+        e.preventDefault();
         setIsLoading(true);
         try {
-            await requestMeetingLink(true);
+            if (freeMeetingLinksEnabled) {
```

Comment:
Unless I'm missing context, this is ambiguously named as "freeMeeting...". Sounds like financially free. Do we mean "freeformMeetingLinks..."?